src/css/grapesjs.scss

```diff
@@ -90,6 +90,54 @@
   right: var(--viewsPanelWidth);
 }
 
+// Sticky CSS tags section
+
+.gjs-pn-views-container > div > div:first-child > div:first-child {
+  position: sticky;
+  top: 0;
+  border-bottom: solid $lighterPrimaryColor 1px;
+  background-color: $primaryColor;
+  box-shadow: 0 0 15px 0 #00000060;
+  z-index: 999;
+}
+
+.gjs-clm-label-sel {
+  opacity: 0.5;
+}
+
+.gjs-clm-sels-info {
+  padding: 7px 0;
+  margin: 0;
+  overflow-x: scroll;
+}
+
+.gjs-clm-sel-rule {
+  word-break: break-all;
+}
+
+#gjs-clm-tags-field.gjs-field {
+  margin: 0;
+}
+
+.gjs-clm-tags {
+  padding-top: 7px;
+  padding-bottom: 0;
+}
+
+#gjs-clm-up {
+  margin-top: 0;
+}
+
+// Adapting the layout of the suggestion list
+
+.gjs-suggest {
```

Comment:
[this is a plugin, check its doc](https://github.com/silexlabs/grapesjs-ui-suggest-classes/)

You can choose to add this css in the plugin config - this is a good solution if it doesn't mean too much code and do not override the plugins' styles that we want to keep (i am not a fan of these styling options)
https://github.com/silexlabs/Silex/blob/56ccba435e553880e94f0127d56271af0c230f3c/src/ts/client/grapesjs/index.ts#L271

Or add the css in the `grapesjs-plugins.scss` file
https://github.com/silexlabs/Silex/blob/56ccba435e553880e94f0127d56271af0c230f3c/src/css/grapesjs-plugins.scss

Reply:
Thank you. I will add the CSS in the grapesjs-plugins.scss file, it seems to be the cleaner way to do it.